airbyte-integrations/connectors/source-okta/source_okta/source.py

```diff
@@ -180,9 +184,11 @@ def request_params(
         # The log stream use a different params to get data
         # https://developer.okta.com/docs/reference/api/system-log/#datetime-filter
         stream_state = stream_state or {}
-        params = OktaStream.request_params(self, stream_state, stream_slice, next_page_token)
+        params = IncrementalOktaStream.request_params(self, stream_state, stream_slice, next_page_token)
```

Comment:
Can't we just use `self` here instead of `IncrementalOktaStream` ?

Reply:
Then the best option here is to use `super()`

Reply:
done